charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -298,6 +298,7 @@ data:
     prometheus:
       enabled: true
       host: {{ printf "%s-prometheus" .Release.Name }}
+    updateAirflowCheckEnabled: {{ .Values.houston.updateAirflowCheck.enabled }}
```

Comment:
Should this have a default value set in the astronomer/astronomer/values.yaml file ?

Reply:
https://github.com/astronomer/astronomer/blob/master/charts/astronomer/values.yaml#L186-L188 - This is the location @danielhoherd